src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/indexes/createIndex.svelte

```diff
@@ -74,6 +74,14 @@
             });
         } finally {
             showCreateIndex = false;
+
+            // Redirect to the index page if the index is created from the attribute page
+            // P.S. goto is not working here for some reason
+            if (externalAttribute) {
+                // await goto(window.location.pathname.replace('attributes', 'indexes'), {
```

Comment:
Mmmmm strange, did you try this?  `${base}/console/project-${$page.params.project}/databases/database-${databaseId}/collection-${$collection.$id}/indexes`

Reply:
Yeah, it didn't work, unfortunately 😢